src/main/assembly/dist/cfg/config.yml

```
@@ -73,36 +73,40 @@ sword2:
     minThreads: 2
     nameFormat: reschedule-worker-%d
 
-#
-# Client user accounts.
-#
-users:
+
+authorization:
+  passwordDelegate:
+    url: http://localhost:20000/
+    forwardHeaders:
+      - authorization
+      - x-dataverse-key
 
   #
-  # Example user
+  # Client user accounts.
   #
-  - name: user001
+  users:
 
-    # BCrypt encrypted password (set to the string 'user001')
-    passwordHash: $2a$10$yvmSYczU7z4KL6qmRCTgTeSvo7uurwPUbB9s/mTKzJrYM/sQKgF.y
+    #
+    # Example user
+    #
+    - name: user001
 
-    # The delegate that will perform a check with dataverse to verify credentials
-    # This property is mutually exclusive with passwordHash
-    # passwordDelegate: http://localhost:20340/
+      # BCrypt encrypted password (set to the string 'user001')
+      passwordHash: $2a$10$yvmSYczU7z4KL6qmRCTgTeSvo7uurwPUbB9s/mTKzJrYM/sQKgF.y
```

```suggestion
      # passwordHash: $2a$10$yvmSYczU7z4KL6qmRCTgTeSvo7uurwPUbB9s/mTKzJrYM/sQKgF.y
```
We should not have a user with an easy password by default.